### PR TITLE
ok

### PR DESCRIPTION
**Pull request by s3mant**

### CODE REVIEW

**Review by s3mant — COMMENTED** [review]
looks gud ig

**Comment by MaskDuck** [comment]
you cant review urself :trollface:

File: domains/badboy.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "description": "Bad Boy's Website",
-  "repo": "https://github.com/Bad-Boy-Codes/bad-boy-codes.github.io",
   "owner": {
     "username": "bad-boy-codes",
-    "email": "sonupanda078@gmail.com"
+    "email": "sonupanda078@gmail.com",
+    "discord": "445073800850046977",
```
Comment (MaskDuck) [inline domains/badboy.json:6]:
```suggestion
    "discord": "Bad Boy Plays#9608",
```

File: domains/badboy.json [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+    "twitter": "BadBoyGoesBrr"
```
Comment (MaskDuck) [inline domains/badboy.json:7]:
```suggestion
    "twitter": "BadBoyGoesBrr",
    "note": "My Discord ID is 445073800850046977."
```
optional, if you want to add ur id here then commit this

File: domains/mail.badboy.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "bad-boy-codes",
+    "email": "sonupanda078@gmail.com",
+    "discord": "445073800850046977"
```
Comment (MaskDuck) [inline domains/mail.badboy.json:5]:
```suggestion
    "discord": "Bad Boy Plays#9608",
    "note": "My Discord ID is 445073800850046977."
```

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Review by s3mant — COMMENTED** [review]
everything's fine!

**Comment by Bad-Boy-Codes** [comment]
> you cant review urself :trollface:

if I do 😳

**Review by MaskDuck — APPROVED** [review]
these file should be renamed to baldboy.json and mail.baldboy.json tbh

**Comment by Bad-Boy-Codes** [comment]
> these file should be renamed to baldboy.json and mail.baldboy.json tbh

i have hairs 🥲

**Comment by Bad-Boy-Codes** [comment]
> > these file should be renamed to baldboy.json and mail.baldboy.json tbh
> 
> i have hairs 🥲

as am not a duck 🙂

**Review by hackermondev — APPROVED** [review]

**Pull request merged**